RELEASE.md

```diff
@@ -0,0 +1,12 @@
+New versions of the gem are cut by the Postmark team, this is a quick guide to ensuring a smooth release.
+
+1. Verify all builds are passing on Travis for your branch.
+1. Merge in your branch to master.
+1. Update VERSION and lib/postmark/version.rb with the new version.
+1. Update CHANGELOG.rdoc with a brief description of the changes.
+1. Commit to git with a comment of "Bump version to x.y.z".
+1. run `rake release` - This will push to github(with the version tag) and rubygems with the version in lib/postmark/version.rb.
+  *Note that if you're on Bundler 1.17 there's a bug that hides the prompt for your OTP, if it hangs then it's asking for your OTP*
```

Comment:
Maybe we should make it clear that bundler 2.x is required for this to work correctly?

Reply:
Just confirmed it does work properly on 2.x.

```
➜  postmark-gem git:(master) rake release
postmark 1.21.2 built to pkg/postmark-1.21.2.gem.
Tag v1.21.2 has already been created.
^Crake aborted!
Interrupt:
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/gem_helper.rb:187:in ``'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/gem_helper.rb:187:in `block in sh_with_code'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/shared_helpers.rb:65:in `chdir'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/shared_helpers.rb:65:in `block in chdir'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/shared_helpers.rb:64:in `chdir'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/gem_helper.rb:186:in `sh_with_code'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/gem_helper.rb:175:in `sh'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/gem_helper.rb:102:in `rubygem_push'
/Users/nick/.gem/ruby/2.4.9/gems/bundler-1.17.3/lib/bundler/gem_helper.rb:69:in `block in install'
/Users/nick/.gem/ruby/2.4.9/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => release => release:rubygem_push
(See full trace by running task with --trace)
➜  postmark-gem git:(master) gem uninstall bundler
Gem bundler-2.1.4 cannot be uninstalled because it is a default gem

Select gem to uninstall:
 1. bundler-1.17.3
 2. bundler-2.1.4
 3. All versions
> 1
Successfully uninstalled bundler-1.17.3
➜  postmark-gem git:(master) rake release
postmark 1.21.2 built to pkg/postmark-1.21.2.gem.
Tag v1.21.2 has already been created.
Pushing gem to https://rubygems.org...
You have enabled multi-factor authentication. Please enter OTP code.
Code:   ^CERROR:  Interrupted
rake aborted!
Interrupt:
/Users/nick/.gem/ruby/2.4.9/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
Tasks: TOP => release => release:rubygem_push
(See full trace by running task with --trace)
```

Reply:
It'll still 'work', it's just clunky. It hangs and if you enter your OTP then it'll proceed, it's just missing the prompt.